### PR TITLE
add psp + labels

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

File: helm/eventrouter-app/templates/rbac.yaml [hunk 1]
```diff
@@ -3,20 +3,40 @@ kind: ServiceAccount
...
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "eventrouter-app.fullname" . }} 
+  name: {{ include "eventrouter-app.fullname" . }}
+  labels:
+    chart: {{ template "eventrouter-app.chart" . }}
+    app: {{ template "eventrouter-app.name" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"s
```
Comment (sjentzsch) [inline helm/eventrouter-app/templates/rbac.yaml:39]:
```suggestion
    heritage: "{{ .Release.Service }}"
```

File: helm/eventrouter-app/values.yaml [hunk 2]
```diff
@@ -24,3 +24,10 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+networkPolicies:
+  # when  enabled, it will create default deny-all policy together with an app  specific policy
+  # kubernetes endpoint is required then
+  enabled: falses
```
Comment (omegas27) [inline helm/eventrouter-app/values.yaml:31]:
```suggestion
  enabled: false
```

File: helm/eventrouter-app/values.yaml [hunk 3]
```diff
@@ -24,3 +24,10 @@ nodeSelector: {}
...
+  # provide 'kubectl get endpoints kubernetes' CIDR e.g. 10.158.0.5/32
+  kubernetesEndpointCidr: ''
```
Comment (omegas27) [inline helm/eventrouter-app/values.yaml:33]:
```suggestion
  kubernetesEndpointCIDR: ''
```

File: helm/eventrouter-app/templates/rbac.yaml [hunk 4]
```diff
@@ -3,20 +3,40 @@ kind: ServiceAccount
...
+  labels:
+    chart: {{ template "eventrouter-app.chart" . }}
+    app: {{ template "eventrouter-app.name" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
 rules:
 - apiGroups: [""]
   resources: ["events"]
   verbs: ["get", "watch", "list"]
+- apiGroups: ['extensions']
```
Comment (omegas27) [inline helm/eventrouter-app/templates/rbac.yaml:25]: This should be encapsulated in an if networkPolicies.enabled block

File: helm/eventrouter-app/templates/psp.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+apiVersion: policy/v1beta1
```
Comment (omegas27) [inline helm/eventrouter-app/templates/psp.yaml:1]: I would encapsulate that in an if networkPolicies.enabled block

File: helm/eventrouter-app/templates/app-network-policy.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    heritage: "{{ .Release.Service }}"
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "eventrouter-app.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  egress:
+  - to:
+    - ipBlock:
+        cidr: {{ .Values.networkPolicies.kubernetesEndpointCidr }}
```
Comment (omegas27) [inline helm/eventrouter-app/templates/app-network-policy.yaml:20]:
```suggestion
        cidr: {{ .Values.networkPolicies.kubernetesEndpointCIDR }}
```

**Pull request merged**